velox/expression/CastExpr.cpp

```diff
@@ -247,6 +247,30 @@ VectorPtr CastExpr::castToDate(
   }
 }
 
+VectorPtr CastExpr::castIntToTimestamp(
+    const SelectivityVector& rows,
+    const BaseVector& input,
+    exec::EvalCtx& context) {
```

Comment:
Is the behavior the same between Spark and Presto? If not, we may need to separate the implementation.